nucleus/common/mbeanserver/src/main/java/org/glassfish/admin/mbeanserver/ssl/SecureRMIServerSocketFactory.java

```diff
@@ -45,7 +46,7 @@
 public class SecureRMIServerSocketFactory extends SslRMIServerSocketFactory {
 
     private final InetAddress mAddress;
-    private final ServiceLocator habitat;
+    private final ServiceLocator locator;
```

Comment:
Absolutely tiny remark, and no need to change it, but I think "serviceLocator" as variable name would be more clear (various other locations use that too)

Reply:
We use both, I prefer shorter version as it is clear in the context what it is.